Failure publishing some crates due to missing dep versions

There's been some intermittent publish failures over the last few weeks that's resulted in a couple missing crate versions, most recently with rustc_parse. I'm hoping the issue is just timing with dependency relationships between these crates, and the new dependent version not quite hitting the index before the consuming crate versions are published.

```bash
publishing rustc_fs_util 663.0.0
    Updating crates.io index
   Packaging rustc-ap-rustc_fs_util v663.0.0 (/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/foo.bixavKjKmtxA/rust-ccac43b86b559e01fa71179af1a838ab94559c75/src/librustc_fs_util)
   Uploading rustc-ap-rustc_fs_util v663.0.0 (/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/foo.bixavKjKmtxA/rust-ccac43b86b559e01fa71179af1a838ab94559c75/src/librustc_fs_util)
publishing rustc_session 663.0.0
    Updating crates.io index
   Packaging rustc-ap-rustc_session v663.0.0 (/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/foo.bixavKjKmtxA/rust-ccac43b86b559e01fa71179af1a838ab94559c75/src/librustc_session)
   Uploading rustc-ap-rustc_session v663.0.0 (/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/foo.bixavKjKmtxA/rust-ccac43b86b559e01fa71179af1a838ab94559c75/src/librustc_session)
publishing rustc_parse 663.0.0
    Updating crates.io index
error: failed to select a version for the requirement `rustc-ap-rustc_session = "^663.0.0"`
  candidate versions found which didn't match: 662.0.0, 660.0.0, 659.0.0, ...
  location searched: crates.io index
required by package `rustc-ap-rustc_parse v663.0.0 (/private/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/foo.bixavKjKmtxA/rust-ccac43b86b559e01fa71179af1a838ab94559c75/src/librustc_parse)`
thread 'main' panicked at 'assertion failed: result.success()', src/main.rs:380:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
##[error]Process completed with exit code 101.
```

--- a/src/main.rs
+++ b/src/main.rs
@@ -56,7 +56,7 @@ fn main() {
         publish(p, &commit, &version_to_publish);
 
         // Give the crates time to make their way into the index
-        thread::sleep(Duration::from_secs(10));
+        thread::sleep(Duration::from_secs(20));
     }
 }
 

